src/test/scala/com/fulcrumgenomics/fastq/DemuxFastqsTest.scala

```diff
@@ -421,6 +421,72 @@ class DemuxFastqsTest extends UnitSpec with OptionValues with ErrorLogLevel {
     }
   }
 
+  Seq(true, false).foreach { illuminaStandards =>
+    // A file containing valid paired-end FASTQ records.
+    val pairedFastqPath = {
+      val fastqs = new ListBuffer[FastqRecord]()
+      fastqs += fq(name="Instrument:RunID:FlowCellID:Lane:Tile:X:1 ReadNum:FilterFlag:0:SampleNumber", bases=sampleBarcode1 + "A"*100) // matches the first sample -> first sample
```

Comment:
It's a little weird to include the name and the comment via `name=`.  If you added a read number here it would do the wrong thing and write `Instrument:RunID:FlowCellID:Lane:Tile:X:1 ReadNum:FilterFlag:0:SampleNumber/1`.

Maybe add `(..., comment: Option[String] = None)` to the `fq` method and use it?